crypto/rsa/rsa_gen.c

```diff
@@ -72,16 +72,6 @@ static int rsa_builtin_keygen(RSA *rsa, int bits, int primes, BIGNUM *e_value,
     BN_CTX *ctx = NULL;
     BN_ULONG bitst = 0;
 
-    /*
-     * From Github pull request #4241:
-     *
-     * We are in disagreement on how to handle security trade-off, in other
```

Comment:
Has this disagreement been resolved?

Reply:
Well, view this request as attempt to resolve it. On related note I'd argue that the degree of misconception and misunderstanding from discussion participants speaks only in favour of coding the constraints. I mean if we can't get it right, what chances does user have?

Reply:
This is also one of my worries. I would really like to avoid users generating keys that are weak without realizing it, and I'd rather be on the conservative side.